schemas/v1/25_fevm_contract.go

```diff
@@ -0,0 +1,19 @@
+package v1
+
+func init() {
+	patches.Register(
+		25,
+		`
+	CREATE TABLE IF NOT EXISTS {{ .SchemaName | default "public"}}.fevm_contract  (
+	    height               BIGINT NOT NULL,
+		actor_id             TEXT,
+		eth_address          TEXT,
+		byte_code            TEXT,
+		byte_code_hash       TEXT,
+		balance              numeric,
+		nonce                BIGINT,
+		PRIMARY KEY(height, actor_id)
```

Comment:
Did Starboard also have the PK on the `height` and `actor_id`? We might also want to add another column to that, either the `eth_address` or `nonce` as an actor can deploy many contracts.